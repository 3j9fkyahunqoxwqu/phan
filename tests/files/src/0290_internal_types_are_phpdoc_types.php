<?php

$result = filter_id('not-a-filter');
$result->doSomething();
