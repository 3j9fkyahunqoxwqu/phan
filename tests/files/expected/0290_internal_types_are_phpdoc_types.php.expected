%s:4 PhanNonClassMethodCall Call to method doSomething on non-class type bool|int
